src/main/java/com/snowflake/kafka/connector/internal/LoggerHandler.java

```diff
@@ -0,0 +1,123 @@
+package com.snowflake.kafka.connector.internal;
+
+import com.snowflake.kafka.connector.Utils;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoggerHandler {
+  // static properties and methods
+  private static final UUID CORRELATION_ID_EMPTY = null;
+  private static final Logger META_LOGGER = LoggerFactory.getLogger(LoggerHandler.class.getName());
+  private static UUID LOGGER_CORRELATION_ID = CORRELATION_ID_EMPTY;
```

Comment:
umm, ideally, since this is not static final, it shouldnt be all uper case. should we instead use `loggerCorrelationId`?